common/cards/effects/berry-bush.ts

```diff
@@ -0,0 +1,114 @@
+import EffectCard from '..//base/effect-card'
+import {isTargetingPos} from '../../utils/attacks'
+import {GameModel} from '../../models/game-model'
+import {discardCard} from '../../utils/movement'
+import {CardPosModel, getBasicCardPos} from '../../models/card-pos-model'
+import {TurnActions} from '../../types/game-state'
+
+class BerryBushEffectCard extends EffectCard {
+	constructor() {
+		super({
+			id: 'berry_bush',
+			numericId: 173,
+			name: 'Sweet Berry Bush',
+			rarity: 'ultra_rare',
+			description:
+				"Use like a Hermit card. Place on one of your opponent's empty Hermit slots. Has 30hp.\n\nCan not become active. Can not attach cards to it.\nYou do not get a point when it's knocked out.\n\nLoses 10hp per turn. If you knock out Sweet Berry Bush before it's HP becomes 0, add 2 Instant Healing II into your hand.",
+		})
+	}
+
+	override onAttach(game: GameModel, instance: string, pos: CardPosModel) {
+		const newPos = getBasicCardPos(game, instance)
+		if (!newPos) return
+		const {player, opponentPlayer, row, rowIndex} = newPos
+		if (!row) return
+
+		row.health = 30
+		if (player.board.activeRow === null) {
```

Comment:
The description states that it can't be changed to be active, but that's exactly what this if statement does